py/server/tests/test_parquet.py

```diff
@@ -51,15 +49,15 @@ def test_crd(self):
         with self.subTest(msg="write_table(Table, str)"):
             write(table, file_location)
             self.assertTrue(os.path.exists(file_location))
-            table2 = read(file_location)
+            table2 = read(file_location, type=ParquetType.SINGLE)
```

Comment:
These type specs are not needed, correct?  If they are needed, I'm sure user code will break.

Reply:
This is mostly about having the test code be more targeted - no reason to test general parquet layout inference for every single test IMO. There are tests specifically designed to test layout inference.

Reply:
I'm good with that.  Just confirming that we weren't breaking users.